bindgen/codegen/mod.rs

```diff
@@ -4133,6 +4148,18 @@ impl CodeGenerator for Function {
                 quote! { #[link(wasm_import_module = #name)] }
             });
 
+        if is_internal && ctx.options().wrap_non_extern_fns {
+            let name = canonical_name.clone() +
+                ctx.options()
```

Comment:
Maybe let's add a `ctx.non_extern_fns_suffix()` that does this `as_deref(...).unwrap_or()`? There's some duplication otherwise, but no huge deal.